assets/src/design-system/theme/colors.js

```diff
@@ -253,6 +256,9 @@ const lightTheme = {
   form: {
     dropDownSubtitle: brand.gray[50], // equivalent to fg.tertiary
   },
+  shadows: {
```

Comment:
nit: can this be singular not plural? (shadow not shadows) since the rest of the keys are singular.